src/main/java/mate/academy/spring/model/dto/request/UserRequestDto.java

```diff
@@ -0,0 +1,22 @@
+package mate.academy.spring.model.dto.request;
+
+public class UserRequestDto {
+    private String email;
+    private String password;
+
+    public String getEmail() {
+        return email;
+    }
```

Comment:
We could add here validation annotations:
```suggestion
   @NotNull
    @Email
    private String email;
    @NotNull
    private String password;

    public String getEmail() {
        return email;
    }
```